modules/cli/pages/cli-intro.adoc

```diff
@@ -39,46 +39,117 @@ You can collect diagnostics through the command-line interface by using the xref
+* Windows: https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-admin-tools_{tools-ver}-windows_amd64.zip[]
 
 Unzip or untar the packages, and the binaries are ready to run.
 For example:
 
 [source,console,subs="attributes+"]
 ----
-wget https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-tools_{tools-ver}-linux_x86_64.tar.gz
+wget https://packages.couchbase.com/releases/{tools-ver}/couchbase-server-admin-tools_{tools-ver}-linux_x86_64.tar.gz
 
-tar -xf couchbase-server-tools_{tools-ver}-linux_x86_64.tar.gz
+tar -xf couchbase-server-admin-tools_{tools-ver}-linux_x86_64.tar.gz
 ----
 
 Each package also contains a `README` file and a copy of the software license.
 
 NOTE: On Windows, you must have a recent version of the Microsoft Visual {cpp} Redistributable runtime libraries installed.
 If you do not have these libraries installed, download them from https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170[Microsoft Visual {cpp} Redistributable latest supported downloads].
 
+
 [#version-compatibility]
 == Version Compatibility
 
 The versions of the utilities installed by the tools package are the same as the corresponding Couchbase Server installation package.
 
-The {tools-ver} `cbimport`, `cbexport`, `cbbackupmgr`, and `cbq` utilities are compatible with the following Couchbase Server versions:
+=== For Server Developer Tools Packages
+
+The {tools-ver} version of `cbimport`, `cbexport`, `cbbackupmgr`, `cbq`, `couchbase-cli`, `mctimings`, `cbstats`, and `mcstat` utilities are compatible with the following Couchbase Server versions:
+
+* 7.6.0, 7.6.1, 7.6.2, 7.6.3
+* 7.2.x
+* 7.1.x
+* 7.0.x
+
+The 7.6.2 version of `cbimport`, `cbexport`, `cbbackupmgr`, and `cbq` utilities are compatible with the following Couchbase Server versions:
 
 * 7.6.0, 7.6.1
 * 7.2.x
 * 7.1.x
 * 7.0.x
+
+So, you can download and use the latest version of these utilities (`cbimport`, `cbexport`, `cbbackupmgr`, `cbq`, `couchbase-cli`, `mctimings`, `cbstats`, and `mcstat`) with the previous Couchbase Server versions.
+
+=== For Server Admin Tools Packages
+
+The {tools-ver} version of `cbimport`, `cbexport`, `cbbackupmgr`, `cbq`, `couchbase-cli`, `cbstats`, `cbdatarecovery`, `mctimings`, `mcstat`, and `mctestauth` utilities are compatible with the following Couchbase Server versions:
+
+* 7.6.0, 7.6.1, 7.6.2, 7.6.3
+* 7.2.x
+* 7.1.x
+* 7.0.x 
+
+NOTE: Determine the version of any Server admin tools package utility by using the `--version` option, except for `cbc`. Determine the `cbc` utility version by using `cbc version` because `couchbase-cli` is a utility for managing/administering the Couchbase cluster. So, the version of `couchbase-cli` and the Couchbase cluster must match. For all the Server admin tools package utilities, any existing feature in a higher Server version is not supported by a lower version utility.
```

Comment:
Looks like some of the sentences in this NOTE got spliced together in the wrong places.  The information in the NOTE section is the only information that should be in the Version compatibility section.

- Use the `--version` option to get the version of the utility, except for `cbc`, which uses `cbc version`.
- Because `couchbase-cli` is a utility for administering the Couchbase cluster, the version of `couchbase-cli` and the Couchbase cluster should match.
- For all utilities, a feature that exists in a higher Server version only will not be supported by a lower version utility.

Reply:
This is done.